src/components/Culr/culr.component.js

```diff
@@ -235,7 +234,7 @@ async function createUser(user, agencyId) {
 
   // Check if user has logged in on municipality
   // Create user on CULR.
-  const borchkInfo = await getBorchkInfo(user);
+  const borchkInfo = (user.agency === agencyId) ? await getBorchkInfo(user) : {};
```

Comment:
I line 242 bruger du borchkInfo.municipalityNumber - men du sætter et tomt object her.
Er der ikke risiko for at det fejler?

Reply:
Jovda. Fixed